### DISCUSSION TITLE
SCTE-35 standard version

### DISCUSSION THREAD

**Discussion by enricovittorini** [futzu/threefive#80] [Q&A]
Hello all, 
I would like to know which scte-35 standard is implemented to parse scte35 and if it is possible to decode it by specifying a specific standard. thanks!

**Comment by futzu** [comment]
I am only aware of one SCTE-35 standard for parsing, 
and threefive meets the 2022-b SCTE-35 specification.

Are you talking about different versions of the spec like 2013, and 2017?
If that's what you're talking about, new versions are backward compatible with older versions.

What are you trying to do here?

**Reply by enricovittorini** [reply]
yes i am talking about different versions. something like https://www.middleman.tv/scte35-parser
where i can set the standard version to be used to parse an base64 string

**Reply by futzu** [reply]
That makes no sense at all.

1.  Newer versions of the specification are backwards compatible with older versions. a new parser can parse SCTE=35 created to an earlier specification.

 2. An older parser  can parse SCTE-35 Cues made to a newer specification.  by only parsing the parts that it understands and ignoring anything it doesn't understand. 

 https://www.middleman.tv/scte35-parser has several minor errors in their output, and it is few revisions behind the current specification. 

threefive  is very compatible, it's used by nine of the top ten broadcast companies in the U.S.
